README.md

```diff
@@ -1,2 +1,134 @@
-# terraform-cloud-workspace-action
-A GitHub action for managing Terraform cloud workspaces
+# Terraform Cloud Workspace Action
+
+A GitHub action for managing Terraform Cloud workspaces
+
+## Usage
+
+```yaml
+name: TFLint
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: takescoop/terraform-cloud-workspace-action@v0
+        with:
+          terraform_token: "${{ secrets.TF_TOKEN }}"
+          terraform_organization: "${{ takescoop }}"
+          apply: "${{ github.ref == 'refs/heads/main' }}"
```

Comment:
https://stackoverflow.com/a/68414395

Reply:
```suggestion
          apply: "${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}"
```

Reply:
Try it to be sure. We also have `$default-branch` to use in workflow templates, alternative would just be to use that syntax in this example and have a code comment telling the user to replace it.

Reply:
it works! 🥳